README.md

```diff
@@ -1,6 +1,7 @@
 # Food Delivery App
 ![](banner.png)
-A new Flutter project
+
+The food ordering & delivery application is designed based on the real-life scenario. Anyone can use this code for their own purpose. If you are interested please make your contribution to the code.
```

Comment:
```suggestion
This is a food ordering & delivery application built with a flutter framework and firebase real-time database. 
Anyone can use this code for their own purpose. If you are interested please make your contribution to the code.
```

IMO we should provide information about the framework used to build this application,

What do you say?

Reply:
I agreed